template/panel/install/Config_tpl.txt
<?php
if (! defined ( 'ROOT_DIR' )){ exit ( "Error, wrong way to file.<a href=\"/\">Go to main</a>.");}
/**
* Это главный ключ группы он хранится в админ панели
* This is the main key of the group it is stored in the admin panel.
*/
define('API_KEY',       "%API_KEY%");
/**
* Это URL центрального API
* This is the central API URL.
*/
define('API_URL',       "https://api.mmoweb.ru/");

/**
* Тут настраивается подключение к базе хостинга
* This is where the connection to the hosting database is configured.
*/
define('DB_HOST',       '%DB_HOST%');
define('DB_NAME',       '%DB_NAME%');
define('DB_USER',       '%DB_USER%');
define('DB_PASSWORD',   '%DB_PASSWORD%');

/**
* Прочие настройки
* Other settings
*/
define('DEBUG',          TRUE);
define('DEBUG_IP',       '0.0.0.0');
define('DETECT_LANG',    TRUE);
define('UTF8_ENABLED',   TRUE);
define("VIEWPATH",       '/template');
define("CACHEPATH",      '/cache');


/**
* Готовые шаблоны можно купить в админ панели ммовеб
* Шаблон сайта. Находится: /template/site
* Ready-made templates can be bought in the admin panel of mmoweb
* Site template. Location: /template/site
*/
define('TEMPLATE',       'default'); //По умолчанию: default
define('TEMPLATE_DIR',   VIEWPATH . '/site/'.TEMPLATE); //Sys: полный путь к файлам шаблона

/**
* Настройки обрашений к АПИ
* API Settings
*/
define('CONNECTION_MAX_COUNT',  1);  //MAX Count connection
define('CONNECTION_TIME',       2);  //Sec

/**
* Доступ в панель управления базой данных предметов
* Access to the item database control panel
*/
define('ADMIN_PANEL',       true);
define('ADMIN_LOGIN',       '%ADMIN_LOGIN%');
define('ADMIN_PASSWORD',    '%ADMIN_PASSWORD%');
define('ADMIN_IP',          '*'); //All ip: *
define('ADMIN_URL',          'admin'); // URL - /admin

define('TEMPLATE_WYSIWYG',  true); //Использовать WYSIWYG HTML Editor или textarea
/**
* Продвинутые настройки
* Advanced settings
*/
define('CLOUD_FLARE',        true);         // Переопределение IP адресов от https://www.cloudflare.com/
define('HEADER_IP',         'REMOTE_ADDR'); // Источник ип адреса по умолчанию: REMOTE_ADDR

/**
* Кеширование информации сайт и лк. Время в секундах
* Caching information site and cp. Time in seconds
*/
define('CACHE_NEWS',                    600);
define('CACHE_IBLOCK',                  600);
define('CACHE_FORUM',                   600);
define('CACHE_ONLINE',                  600);
define('CACHE_RATING',                  600);
define('CACHE_HISTORY_ONLINE',          86400);
define('CACHE_STREAM',                  600);
define('CACHE_CLEAR_SESSION',           1800);
#TODO Реализовать очистку временных файлов
define('DEL_CACHE_THROTTLER',           86400);
define('DEL_CACHE_DEBUG',               604800);

/**
 * Визуальная подгрузка предметов из базы данных кабинета
 * Visual loading of items from the cabinet database
 * /template/webdb.js  -- <span class='webdb-item' webdb-id="3936" title="" s-sid="8" s-name="true" s-add="true" s-icon="true" s-descr="true"></span>
 */
define('WEB_ITEM_DB',            true);
//Access-Control-Allow-Origin
define('ACAO_DB_GETAWAY',        '*');

/**
 * Переопределения принятия платежа,
 * функция не стабильна может быть проблемы доставки платежа при атаке на сайт.
 * (Пользоваться только в крайней необходимости)
 *
 * Overrides of payment acceptance,
 * the function is not stable there may be problems with the payment delivery when the site is attacked.
 * (Use only when absolutely necessary)
 */
define('PAYMENT_GATEWAY',   false);
define('PAYMENT_KEY',       'DFCPQ'); //EXEMPLE: https://mw4.mmoweb.ru/api/payment/unitpay/DFCPQ

/**
 * SYS*
 * Для хостеров - если у вас проблемы с заголовками на передачу HTTPS, HTTP_X_FORWARDED_PROTO или HTTP_FRONT_END_HTTPS
 * И происходит постоянный ридирект поставте значение в false
 */
define('HTTP_FORWARDING',   false); //Производить ли ридирект если в админ панели выстовлено https а обрашение на http

/**
 * Plugin Support
 * Настройки плагина суппорта, загрузка файлов
 * Support plugin builders, file upload
 */
define('LIMIT_FILES',   5);
define('MAX_SIZE',      2); //mb


Config.php
<?php
if (! defined ( 'ROOT_DIR' )){ exit ( "Error, wrong way to file.<a href=\"/\">Go to main</a>.");}
/**
* Это главный ключ группы он хранится в админ панели
* This is the main key of the group it is stored in the admin panel.
*/
define('API_KEY',       "q08YmsGbX0lC)54U7ZJ1");
/**
* Это URL центрального API
* This is the central API URL.
*/
define('API_URL',       "https://api.mmoweb.ru/");

/**
* Тут настраивается подключение к базе хостинга
* This is where the connection to the hosting database is configured.
*/
define('DB_HOST',       'localhost');
define('DB_NAME',       'admin_mw4');
define('DB_USER',       'admin_mw4');
define('DB_PASSWORD',   'mVyOIbFVrq');

/**
* Прочие настройки
* Other settings
*/
define('DEBUG',          TRUE);
define('DEBUG_IP',       '0.0.0.0');
define('DETECT_LANG',    TRUE);
define('UTF8_ENABLED',   TRUE);
define("VIEWPATH",       '/template');
define("CACHEPATH",      '/cache');


/**
* Готовые шаблоны можно купить в админ панели ммовеб
* Шаблон сайта. Находится: /template/site
* Ready-made templates can be bought in the admin panel of mmoweb
* Site template. Location: /template/site
*/
define('TEMPLATE',       'default'); //По умолчанию: default
define('TEMPLATE_DIR',   VIEWPATH . '/site/'.TEMPLATE); //Sys: полный путь к файлам шаблона

/**
* Настройки обрашений к АПИ
* API Settings
*/
define('CONNECTION_MAX_COUNT',  1);  //MAX Count connection
define('CONNECTION_TIME',       2);  //Sec

/**
* Доступ в панель управления базой данных предметов
* Access to the item database control panel
*/
define('ADMIN_PANEL',       true);
define('ADMIN_LOGIN',       'admin');
define('ADMIN_PASSWORD',    'admin');
define('ADMIN_IP',          '*'); //All ip: *
define('ADMIN_URL',          'admin'); // URL - /admin

define('TEMPLATE_WYSIWYG',  true); //Использовать WYSIWYG HTML Editor или textarea
/**
* Продвинутые настройки
* Advanced settings
*/
define('CLOUD_FLARE',        true);         // Переопределение IP адресов от https://www.cloudflare.com/
define('HEADER_IP',         'REMOTE_ADDR'); // Источник ип адреса по умолчанию: REMOTE_ADDR

/**
* Кеширование информации сайт и лк. Время в секундах
* Caching information site and cp. Time in seconds
*/
define('CACHE_NEWS',                    600);
define('CACHE_IBLOCK',                  600);
define('CACHE_FORUM',                   600);
define('CACHE_ONLINE',                  600);
define('CACHE_RATING',                  600);
define('CACHE_HISTORY_ONLINE',          86400);
define('CACHE_STREAM',                  600);
define('CACHE_CLEAR_SESSION',           1800);
#TODO Реализовать очистку временных файлов
define('DEL_CACHE_THROTTLER',           86400);
define('DEL_CACHE_DEBUG',               604800);

/**
 * Визуальная подгрузка предметов из базы данных кабинета
 * Visual loading of items from the cabinet database
 * /template/webdb.js  -- <span class='webdb-item' webdb-id="3936" title="" s-sid="8" s-name="true" s-add="true" s-icon="true" s-descr="true"></span>
 */
define('WEB_ITEM_DB',            true);
//Access-Control-Allow-Origin
define('ACAO_DB_GETAWAY',        '*');

/**
 * Переопределения принятия платежа,
 * функция не стабильна может быть проблемы доставки платежа при атаке на сайт.
 * (Пользоваться только в крайней необходимости)
 *
 * Overrides of payment acceptance,
 * the function is not stable there may be problems with the payment delivery when the site is attacked.
 * (Use only when absolutely necessary)
 */
define('PAYMENT_GATEWAY',   false);
define('PAYMENT_KEY',       'DFCPQ'); //EXEMPLE: https://mw4.mmoweb.ru/api/payment/unitpay/DFCPQ

/**
 * SYS*
 * Для хостеров - если у вас проблемы с заголовками на передачу HTTPS, HTTP_X_FORWARDED_PROTO или HTTP_FRONT_END_HTTPS
 * И происходит постоянный ридирект поставте значение в false
 */
define('HTTP_FORWARDING',   false); //Производить ли ридирект если в админ панели выстовлено https а обрашение на http

/**
 * Plugin Support
 * Настройки плагина суппорта, загрузка файлов
 * Support plugin builders, file upload
 */
define('LIMIT_FILES',   5);
define('MAX_SIZE',      2); //mb
